ejercicio1vulky/src/main/java/com/mycompany/ejercicio1vuky/Pepita.java

```diff
@@ -0,0 +1,14 @@
+
+package com.mycompany.ejercicio1vuky;
+
+
+public class Pepita extends Persona{
+    
+    public void tocarFlauta( String cancion){
+        System.out.println("toca"+cancion);            
```

Comment:
Awesome you can implement the same in Pedro class

Reply:
ok